src/Tizen.Peripheral/Tizen.Peripheral/I2c.cs

```diff
@@ -0,0 +1,151 @@
+﻿/*
+* Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using NativeI2c = Interop.Peripheral.I2c;
+
+namespace Tizen.Peripheral
+{
+    /// <summary>
+    /// The class allows applications to communicate via i2c platform's bus.
+    /// </summary>
+    /// <privilege>http://tizen.org/privilege/peripheralio</privilege>
+    public class I2c : IDisposable
+    {
+        /// <summary>
+        /// Native handle to I2c.
+        /// </summary>
+        protected IntPtr _handle;
+
+        private I2c(IntPtr handle)
+        {
+            _handle = handle;
+        }
+
+        /// <summary>
+        /// Opens an I2C slave device.
+        /// </summary>
+        /// <param name="bus">The I2C bus number that the slave device is connected.</param>
+        /// <param name="address">The address of the slave device.</param>
+        /// <returns>The I2c object.</returns>
+        public static I2c Open(int bus, int address)
```

Comment:
This `Open` Method is seen as a factory method.
However, what this method does is not different from what the constructor does.
Even if you don't have to use it as a factory like this, it's not different from just making it in the constructor using the same arguments as this method.

It seems that the factory is not needed, so it is recommended to just use the constructor.